client/www/data/docsNavigation.js

```diff
@@ -41,6 +41,7 @@ module.exports = [
       { title: 'Instant CLI', href: '/docs/cli' },
       { title: 'Storage (beta)', href: '/docs/storage' },
       { title: 'Schema-as-code', href: '/docs/schema' },
+      { title: 'Strong Init (experimental)', href: '/docs/strong-init' },
```

Comment:
Let's change the order so that the beta/experimental things are next to each other
```suggestion
      { title: 'Schema-as-code', href: '/docs/schema' },
      { title: 'Storage (beta)', href: '/docs/storage' },
      { title: 'Strong Init (experimental)', href: '/docs/strong-init' },
```